### ISSUE TITLE
Latest vertx-eventbus-client is 3.5.0

### ISSUE THREAD

**Issue by leolux** [knalli/angular-vertxbus#272]
I am not sure if the following line is outdated: https://github.com/knalli/angular-vertxbus/blob/master/bower.json#L32

There is a explanation of the tilde: https://stackoverflow.com/questions/19541494/bower-dependency-tilde-in-node
I guess that "~3.5.0" satisfies both 3.4.X and 3.5.X but not 3.6.0

**Comment by leolux** [comment]
But there seems to be no changes from 3.4.0 to 3.5.0 regarding releases https://github.com/vert-x3/vertx-bus-bower/releases

**Comment by knalli** [comment]
~3.5.0 is never 3.6
But
~3.5 could be a 3.6

**Comment by leolux** [comment]
So ~3.4.0 does not include 3.5.0?

**Comment by knalli** [comment]
That's the definition of semver, and these parts are in npm und bower the same.

```
$ npx bower install vertx3-eventbus-client@~3.4.0
npx: Installierte 1 in 5.278s
bower vertx3-eventbus-client#~3.4.0           cached https://github.com/vert-x3/vertx-bus-bower.git#3.4.0
bower vertx3-eventbus-client#~3.4.0         validate 3.4.0 against https://github.com/vert-x3/vertx-bus-bower.git#~3.4.0
bower vertx3-eventbus-client#~3.4.0              new version for https://github.com/vert-x3/vertx-bus-bower.git#~3.4.0
bower vertx3-eventbus-client#~3.4.0          resolve https://github.com/vert-x3/vertx-bus-bower.git#~3.4.0
bower vertx3-eventbus-client#~3.4.0         download https://github.com/vert-x3/vertx-bus-bower/archive/3.4.2.tar.gz
bower vertx3-eventbus-client#~3.4.0          extract archive.tar.gz
bower vertx3-eventbus-client#~3.4.0         resolved https://github.com/vert-x3/vertx-bus-bower.git#3.4.2
bower vertx3-eventbus-client#~3.4.0          install vertx3-eventbus-client#3.4.2

vertx3-eventbus-client#3.4.2 bower_components/vertx3-eventbus-client
```

**Comment by leolux** [comment]
3.5.0 has already been released. At the moment the version resolved by bower results in 3.4.2 but that is not a real issue because there hasn't been any changes between 3.4.2 and 3.5.0 in vertx3-eventbus-client. .
